websocket/polygon.go

```diff
@@ -194,11 +193,20 @@ func (c *Client) Unsubscribe(topic Topic, tickers ...string) error {
 	return nil
 }
 
-func (c *Client) Close() {
+func (c *Client) Close() error {
```

Comment:
this all seems like a bad rebase. it's reverting some changes i made yesterday.

Reply:
When I made my branch I thought I had branched from master but I think I had previously set my origin to point to a different branch. I just quickly fixed it in a new branch and I'll push that instead of rebasing this.